### PR TITLE
added wishlist table to sql

### PR DESCRIPTION
**Pull request by billythakidd04**
for #4 wishlist table to sql build file

### CODE REVIEW

File: db/create_db.sql [hunk 1]
```diff
@@ -1,8 +1,8 @@
-CREATE DATABASE IF NOT EXIST mew_co;
+CREATE DATABASE IF NOT EXISTS mew_co;
 
-USE DATABASE mew_co;
+USE mew_co;
```
Comment (rbaia28) [inline db/create_db.sql:3]: why did you remove the declaration?

Comment (billythakidd04) [reply db/create_db.sql:3]: It's a typo I make every single time. The correct command is actually `Use <db_name>`. Using the command there is a syntax error.

**Review by rbaia28 — APPROVED** [review]
